openvdb/points/AttributeArrayString.h

```diff
@@ -192,6 +194,10 @@ class OPENVDB_API StringAttributeWriteHandle : public StringAttributeHandle
     /// Reset the value cache from the metadata
     void resetCache();
 
+    AttributeArray& array();
+
+    bool hasIndex(const Name& name) const;
```

Comment:
Perhaps this could follow a stdlib convention here and use the upcoming "contains()" method for checking the key:

```suggestion
    bool contains(const Name& name) const;
```

https://en.cppreference.com/w/cpp/container/unordered_map/contains

Also, these two methods need comment strings.

Reply:
Apologies for the delay in responding to this @danrbailey. I've made the changes you've suggested now. If you have another moment to cast your eye over them it'd be much appreciated.

Reply:
Looks good to me, thanks for making those changes Matt!